### PR TITLE
exit if node is attached to cluster and ask to run detach-node

### PR DESCRIPTION
**Pull request by devidask27**
## ISSUE TYPE

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that may cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## IMPACTED FEATURES/COMPONENTS:

decommission-node

#### Manual

**Ran decommisssion-node on node which is attached to cluster**
```
root@test-pf9-bare-os-u20-2793249-610-2:~/pf9/bin# pf9ctl decommission-node
New version found. Please upgrade to the latest version
✓ Loaded Config Successfully
Node is connected to vgi-docker-testclus cluster
2023-08-16T05:12:29.2157Z	FATAL	Please run detach-node command first and wait for node to completely remove from cluster and then run decommision-node command
```
#### Reviewers

/cc @psarwate @sudhakarg @NeelavaChatterjee

### CODE REVIEW

File: pkg/pmk/decomissionNode.go [hunk 1]
```diff
@@ -132,6 +132,7 @@ func DecommissionNode(cfg *objects.Config, nc objects.NodeConfig, removePf9 bool
 		} else {
 			//detach node from cluster
 			fmt.Printf("Node is connected to %s cluster\n", nodeInfo.ClusterName)
+			zap.S().Fatalf("Please run detach-node command first and wait for node to completely remove from cluster and then run decommision-node command")
```
Comment (NeelavaChatterjee) [inline pkg/pmk/decomissionNode.go:135]: Can we add a comment here that the rest of the code will be ignored after this call as we are exiting here. Maybe add the JIRA as well why we are doing this. And that we will decide what to do further based on this JIRA - [PMK-5938](https://platform9.atlassian.net/browse/PMK-5938)

Comment (NeelavaChatterjee) [inline pkg/pmk/decomissionNode.go:135]:
```suggestion
			zap.S().Fatalf("Please run detach-node command first and wait for the node to be completely detached(removed) from the cluster and only then run decommision-node command")
```

Comment (NeelavaChatterjee) [reply pkg/pmk/decomissionNode.go:135]: We could also comment out the rest of the code for better visibility that the code is ignored.

Comment (sudhakarg) [reply pkg/pmk/decomissionNode.go:135]: +1 to rephrase.

Comment (sudhakarg) [reply pkg/pmk/decomissionNode.go:135]: Agreed.

**Review by sudhakarg — APPROVED** [review]
lgtm

**Review by NeelavaChatterjee — APPROVED** [review]
/lgtm

**Pull request merged**